src/App.elm

```diff
@@ -106,20 +108,15 @@ renderCheckbox name isChecked =
         , text name
         ]
 
-
 renderGenreFilter : List Genre -> Html.Html Msg
 renderGenreFilter genreFilter =
+    let
```

Comment:
Perfectly understand that. In that case I'd maybe rather factor out that function instead of using lambda for the sake of clean expression/less noise. 🙂

Reply:
If you'd like it, I can push the update?